schema/request-representation.yaml

```diff
@@ -0,0 +1,58 @@
+$schema: http://json-schema.org/draft-04/schema
+title: Panda Request Representation
+description: >
+  A standard format for expressing all of the dimensions of an HTTP request in
```

Comment:
Two reasons for schematizing this, at least.

1. Documentation so Lambda handler authors know exactly what they can expect to receive.

2. In case we want to validate the format of the object coming into the Lambda handler.